src/Web/Server/ConfigureServices.cs

```diff
@@ -0,0 +1,17 @@
+using FluentValidation.AspNetCore;
+
+using MRA.AssetsManagement.Infrastructure.Data;
+
+namespace MRA.AssetsManagement.Web.Server;
+
+public static class ConfigureServices
```

Comment:
Delete this file
---------------
services.AddFluentValidationAutoValidation(); should be registered in Application layer